cmd/snap/cmd_info_test.go

```diff
@@ -120,7 +120,7 @@ func (s *infoSuite) TestInfoPriced(c *check.C) {
 	c.Assert(rest, check.DeepEquals, []string{})
 	c.Check(s.Stdout(), check.Equals, `name:      hello
 summary:   GNU Hello, the "hello world" snap
-publisher: Canonical✓
+publisher: Canonical*
```

Comment:
have we kept a test for the other situation?

Reply:
There's a test with `--unicode=always`, and there's a test that `--unicode=auto` does the right thing